languages/elixir/config.toml

```diff
@@ -7,6 +7,7 @@ brackets = [
     { start = "{", end = "}", close = true, newline = true },
     { start = "[", end = "]", close = true, newline = true },
     { start = "(", end = ")", close = true, newline = true },
+    { start = "\"\"\"", end = "\n\"\"\"", close = true, newline = true, not_in = ["string", "comment"] },
```

Comment:
Was checking the extension to add this!

This is missing the (admittedly less used in my experience) single quote multi line strings btw

```ex
'''
These ones
'''
```

Reply:
The single quote syntax is not for strings but for charlists. https://hexdocs.pm/elixir/syntax-reference.html#charlists

But this is deprecated syntax and instead sigil c must be used.

```elixir
~c"""
My multiline charlist
"""
```

Reply:
Oh my bad I misspoke that it's for strings! Didn't notice it got deprecated since I rarely use it just noticed it was missing 😅